src/axom/primal/geometry/BoundingBox.hpp

```diff
@@ -438,7 +439,9 @@ template <typename T, int NDIMS>
 template <typename OtherT>
 bool BoundingBox<T, NDIMS>::contains(const BoundingBox<OtherT, NDIMS>& otherBB) const
 {
-  return this->contains(otherBB.getMin()) && this->contains(otherBB.getMax());
+  return otherBB.isValid()
+    ? this->contains(otherBB.getMin()) && this->contains(otherBB.getMax())
+    : true;
```

Comment:
Was this the actual bugfix?

Reply:
I added this to make it easier to consistently test the results of `clip(Triangle, BoundingBox) -> Polygon`.

Specifically, the resulting clipped polygon should always be contained by the bounding box, even if the result is an empty polygon. See the second test at the end of this snippet:

https://github.com/LLNL/axom/blob/c4a3a62f1b6e2b30f825516ce29a0073c3edd930/src/axom/primal/tests/primal_clip.cpp#L138-L160